Extension/package.json

```diff
@@ -2540,6 +2540,11 @@
             ]
           },
           "scope": "resource"
+        },
+        "C_Cpp.shortcutMenuBar.buildAndDebugFile": {
```

Comment:
To ship this as an A/B experiment, we need to add:
```
"tags": [
     "experimental"
]
```

to this setting object (see [example in Python's package.json](https://github.com/microsoft/vscode-python/blob/e1773fc5c1bb3201bb4fe279c4850d3920ab1833/package.json#L554)) and change the default value to be false.

Then, I can override the setting's value in Control Tower to be true for the treatment group.

Reply:
For this branch, I haven't applied this change, for testing purposes, but for shipping, I will apply it.